github/helpers/githubActivityGraphURL.ts

```diff
@@ -1,4 +1,4 @@
 
 export function githubActivityGraphUrl(username : string): string {
-    return `https://activity-graph.herokuapp.com/graph?username=${username}&bg_color=ffffff&color=708090&line=24292e&point=24292e`
+    return `https://github-readme-activity-graph.cyclic.app/graph?username=${username}&bg_color=ffffff&color=708090&line=24292e&point=24292e&area=true&hide_border=true`
 }
```

Comment:
@chinma-yyy  can we do something like this to make the params more readable or I am sure you can come with a neater way which can append the params. Would improve readability. 

```

    let base_url = new URL("https://github-readme-activity-graph.cyclic.app/graph");

    base_url.searchParams.append('user_name',username);
    base_url.searchParams.append('bg_color','ffffff');
    base_url.searchParams.append('color','708090');
    base_url.searchParams.append('line','24292e');
    base_url.searchParams.append('area','true');
    base_url.searchParams.append('hide_border','true');
    base_url.searchParams.append('&point','24292e');
    
    return base_url.toString()
```

Reply:
Yeah sure